src/server/zset_family_test.cc

```diff
@@ -578,4 +578,54 @@ TEST_F(ZSetFamilyTest, BlockingTimeout) {
   EXPECT_THAT(resp0, ArgType(RespExpr::NIL_ARRAY));
 }
 
+TEST_F(ZSetFamilyTest, ZDiffError) {
```

Comment:
Beyond this simple test is there any `end to end` or benchmark test I should be aware of/add?